tools/wpt/browser.py

```diff
@@ -93,48 +96,64 @@ def platform_string_geckodriver(self):
     def install(self, dest=None, channel="nightly"):
         """Install Firefox."""
 
-        branch = {
-            "nightly": "mozilla-central",
-            "beta": "mozilla-beta",
-            "stable": "mozilla-stable"
-        }
-        scraper = {
-            "nightly": "daily",
-            "beta": "release",
-            "stable": "release"
+        import mozinstall
+
+        product = {
+            "nightly": "firefox-nightly-latest-ssl",
+            "beta": "firefox-beta-latest-ssl",
+            "stable": "firefox-beta-latest-ssl"
```

Comment:
@jgraham I think what you really want here for stable is https://download.mozilla.org/?product=firefox-latest-ssl&os=osx&lang=en-US. Otherwise you aren't using the release but beta version on stable.